data/humans/occupations.json

```diff
@@ -942,6 +946,7 @@
 		"upholsterer",
 		"urban planner",
 		"usher",
+    "ux designer",
```

Comment:
Shall we put this uppercase for clarity? People can convert to lower case later if they like. 
```suggestion
    "UX designer",
```